sdks/python/installation.mdx

```diff
@@ -0,0 +1,196 @@
+---
+title: Installation
+description: In this section we look at the various options that are available for installing the Tilebox python client.
+icon: box
+---
+
+import { PythonPackageOverview, PipInstall, PoetryInstall, PipenvInstall } from "@/components/PythonInstall";
+
+# Installation
```

Comment:
@lukasbindreiter Feel free to update the installation guide. The introduction page will also need to be updated